bundled_scripts/ilo.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
```

Comment:
do we plan to test these scripts (on some sample output) from management? We have some tests for it in Ralph2: https://github.com/allegro/ralph/tree/develop/src/ralph/scan/tests/plugins (other thing is how to run these tests here).

Btw we could do some end-to-end testing for calling `ralph-cli`, mocks response from the management (or just mock script output), and see how data is changed in ralph web (it should be easy to setup on travis using docker). Of course not in this PR, I'm just thinking about it :)